pkg/runtime/cluster_pool_assignment_provider.go

```diff
@@ -0,0 +1,24 @@
+package runtime
+
+import (
+	"github.com/flyteorg/flyteadmin/pkg/runtime/interfaces"
+
+	"github.com/flyteorg/flytestdlib/config"
+)
+
+const clusterPoolsKey = "cluster_pools"
```

Comment:
Dont we use camel case every where for the keys. Would be good to keep this consistent with other configs

Reply:
we're inconsistent :(
see https://github.com/flyteorg/flyteadmin/blob/master/flyteadmin_config.yaml#L165 and https://github.com/flyteorg/flyteadmin/blob/master/flyteadmin_config.yaml#L192

but I think the convention leans to camel case, updated